libs/components/code-examples/src/lib/modules/forms/radio/help-key/example.component.html

```diff
@@ -1,28 +1,26 @@
-<div class="sky-padding-even-lg">
```

Comment:
Removing these wrappers so that the demos don't have excessive padding on the docs site.